imgcat/imgcat_test.go

```diff
@@ -22,7 +22,10 @@ import (
 )
 
 func ExampleNew() {
-	w := imgcat.New(os.Stdout, imgcat.Width(imgcat.Pixels(100)), imgcat.Inline(true), imgcat.Name("smiley.png"))
+	w, err := imgcat.New(os.Stdout, imgcat.Width(imgcat.Pixels(100)), imgcat.Inline(true), imgcat.Name("smiley.png"))
```

Comment:
Isn't filename a required? I'd prefer to see it as a argument rather than option.

Reply:
it's not required, actually
most of the time I don't set it up tbh

Reply:
Ah true, you pipe the entire image to the writer. That's why you return a write closer. Got it.